nhais/common/request/healthcheck_handler.py

```diff
@@ -0,0 +1,24 @@
+import tornado.web
```

Comment:
Can we refactor this from mhs common into integration-adaptors-common and reuse?